spaceship/lib/spaceship/connect_api/provisioning/provisioning.rb

```diff
@@ -84,9 +84,13 @@ def delete_bundle_id_capability(bundle_id_capability_id:)
         # certificates
         #
 
-        def get_certificates(filter: {}, includes: nil, limit: nil, sort: nil)
+        def get_certificates(profile_id: nil, filter: {}, includes: nil, limit: nil, sort: nil)
```

Comment:
Looks good to me. @joshdholtz any idea, if this could break any existing workflows? (should not, right?)